text/0107-xcm-execution-hints.md

```diff
@@ -0,0 +1,93 @@
+# RFC-0107: XCM Execution hints
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Start Date**  | 23 July 2024                                                                                |
+| **Description** | Add a mechanism for configuring particular XCM executions                                   |
+| **Authors**     | Francisco Aguirre                                                                           |
+
+## Summary
+
+A previous XCM RFC (https://github.com/polkadot-fellows/xcm-format/pull/37) introduced a `SetAssetClaimer` instruction.
+This idea of instructing the XCVM to change some implementation-specific behavior is useful.
+In order to generalize this mechanism, this RFC introduces a new instruction `SetExecutionHints`
+and makes the `SetAssetClaimer` be just one of many possible execution hints.
+
+## Motivation
+
+There is a need for specifying how certain implementation-specific things should behave.
+Things like who can claim the assets or what can be done instead of trapping assets.
+Another idea for a hint:
+- `LeftoverAssetsDestination`: for depositing leftover assets to a destination instead of trapping them
+
+## Stakeholders
+
+- Runtime devs
+- Wallets
+- dApps
+
+## Explanation
+
+A new instruction, `SetExecutionHints`, will be added.
```

Comment:
This may be subjective, though I think `SetHints` sounds a bit more concise, without sacrificing meaning.

Reply:
Sounds good